CHANGELOG.md

```diff
@@ -7,6 +7,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
```

Comment:
I think there should be a caption 1.2.0 in between Unreleased and 1.1.10 right?

Reply:
Ouch, sorry. I'm used to automating this with a Maven plugin for another project I'm working on. I'll add the header.